api/index.php

```diff
@@ -137,13 +137,17 @@
     $data = $request->getParsedBody();
     $db = getConnection();
     $imagen_existe = !empty($_FILES['imagen']['name']);
+    list($width, $height) = getimagesize($_FILES['imagen']['tmp_name']);
 
     if (empty($data['titulo']) || empty($data['precio']) || empty($data['descripcion'])) {
         $success = false;
         $msg = 'Por favor rellene todos los campos requeridos';
     } elseif ($imagen_existe && !in_array($_FILES['imagen']['type'], ['image/jpeg', 'image/jpg'])) {
         $success = false;
         $msg = 'Solo se permiten imagenes con formato jpg';
+    } elseif (($width < $height) == true) {
```

Comment:
you had to add the $imagen_exist here too to enter this validation

Reply:
acabo de repararlo 🙏